middleware/bodyValidation.js

```diff
@@ -0,0 +1,66 @@
+const { body, validationResult } = require('express-validator');
+
+//Register Form
+const registerRules = () => [
```

Comment:
This doesn't need to be a function, you can have arrays of middleware in a express route handler, so for example you could simply do...
```js
const registerRules = [
  body('name', 'Name is required ').notEmpty(),
  body('email', 'Please include a valid email').isEmail(),
  body(
    'password',
    'Please enter a password with 6 or more characters'
  ).isLength({
    min: 6,
    max: 20
  })
];
```
No need for a function.

Reply:
Yep you're right , unnecessary functions  , thx for you reply , i will fix that  :D

Reply:
No worries. I do like the idea but yeah I don't think we _need_ this.